src/applications/vaos/services/mocks/index.js

```diff
@@ -220,7 +224,9 @@ const responses = {
   },
   'GET /vaos/v2/appointments/:id': (req, res) => {
     const appointments = {
-      data: requestsV2.data.concat(require('./v2/confirmed.json').data),
+      data: requestsV2.data
+        .concat(require('./v2/confirmed.json').data)
```

Comment:
I know it's not a core part of the changes in this PR but can we put this in a variable at the top of the file like the other json data? It looks like it's just the v2 confirmed.json here and in the `GET` above this one.